test/e2e/arktos/multi_tenancy/test_suites/kubectl/test_set.yaml

```diff
@@ -0,0 +1,114 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Kubectl set Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies command "kubectl set".
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_tenant: random_8
+  test_ns: random_8
+  sa_name: e2etest
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${test_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${test_tenant} created\n"
+
+  - Command: ${kubectl} create namespace ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "namespace/${test_ns} created\n"
+
+  - Command:  ${kubectl} apply -f ${test_data_dir}/sample-deployment.yaml -n ${test_ns} --tenant ${test_tenant} 
+    OutputShouldBe: "deployment.apps/sample-nginx-deployment created\n"
+
+###########################################################################################################
+# test "kubectl set image"
+###########################################################################################################
+  - BeforeTestMessage: "Verifying \"kubectl set image\" ..."
+    Command: "${kubectl} get deployment sample-nginx-deployment -n ${test_ns} --tenant ${test_tenant} -o json
+            | jq .spec.template.spec.containers[0].image "
+    OutputShouldBe: "\"nginx:1.7.9\"\n"
```

Comment:
also here about 1.7.9

Reply:
same as above. No. the value of 1.7.9 is from the test data file we set.